core/common/buffer.cpp

```diff
@@ -39,6 +39,10 @@ namespace kagome::common {
     return hex_lower(data_);
   }
 
+  std::string Buffer::toString() const {
+    return std::string(reinterpret_cast<const char*>(data_.data()), data_.size()); // NOLINT
```

Comment:
Perhaps, better to return **std::string_view**. It helps to avoid cteation intermediate string

Reply:
Of course, good catch. Fixed and rebased.